setuptools/_distutils/command/bdist_rpm.py

```diff
@@ -120,6 +120,10 @@ class bdist_rpm(Command):
         ('force-arch=', None,
          "Force an architecture onto the RPM build process"),
 
+        # Allow a packager to explicitly force a version
+        ('force-version=', None,
+         "Force a version onto the RPM instead of using metadata"),
+
```

Comment:
Oh, dear. I'm sorry I just noticed, but I see now this change is applied to `_distutils`, which is currently maintained in a separate repo at [/pypa/distutils](/pypa/distutils). Can you contribute the change there first?

Reply:
On second thought... let me see if I can cherry-pick this commit into that repo and save you the trouble of re-implementing it.